var/spack/repos/builtin/packages/espressopp/package.py

```diff
@@ -0,0 +1,80 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Espressopp(CMakePackage):
+    """ESPResSo++ is an extensible, flexible, fast and parallel simulation
+       software for soft matter research. It is a highly versatile software
+       package for the scientific simulation and analysis of coarse-grained
+       atomistic or bead-spring models as they are used in soft matter research
+    """
+    homepage = "https://espressopp.github.io"
+    url      = "https://github.com/espressopp/espressopp/tarball/v1.9.4"
+
+    version('develop', git='https://github.com/espressopp/espressopp.git', branch='master')
+    version('1.9.4', 'f2a27993a83547ad014335006eea74ea')
+
+    variant('debug', default=False, description='Build debug version')
+    variant('ug', default=False, description='Build user guide')
+    variant('pdf', default=False, description='Build user guide in pdf format')
+    variant('dg', default=False, description='Build developer guide')
+
+    depends_on("cmake@2.8:", type='build')
+    depends_on("mpi", type=('build', 'link', 'run'))
+    depends_on("boost+serialization+filesystem+system+python+mpi", when='@1.9.4:', type=('build', 'link', 'run'))
+    extends("python")
+    depends_on("python@2:2.7.13")
+    depends_on("py-mpi4py@2.0.0", when='@1.9.4:', type=('build', 'link', 'run'))
+    depends_on("py-mpi4py@1.3.1:", when='@develop', type=('build', 'link', 'run'))
+    depends_on("fftw", type=('build', 'link', 'run'))
+    depends_on("py-sphinx", when="+ug", type='build')
+    depends_on("py-sphinx", when="+pdf", type='build')
+    depends_on("texlive", when="+pdf", type='build')
+    depends_on("doxygen", when="+dg", type='build')
+
+    run_tests = True
```

Comment:
You probably don't want this in here. Tests are buggy depending on what OS you are on. For example, if someone wants to build `espressopp` on bgq, the tests may not work. If you hard-code `run_test = True` in here, what you're saying is you cannot install the package unless the tests pass. This may be fine for you, but it may not be fine for other users. I would remove this line. You can specify it on the command line when you install:
```
spack install --run-tests espressopp
```